app-arch/engrampa/engrampa-1.28.2.ebuild

```diff
@@ -3,11 +3,8 @@
 
 EAPI=8
 
-if [[ ${PV} == 9999* ]]; then
-	EGIT_REPO_URI="https://github.com/mate-desktop/${PN}.git"
-	inherit git-r3
-else
-	SRC_URI="https://github.com/mate-desktop/${PN}/archive/${P}.tar.xz"
+MINOR=$(($(ver_cut 2) % 2))
+if [[ ${MINOR} -eq 0 ]]; then
```

Comment:
58296888491e6c6bc8b1f79c45150888d6d722c4

```diff
commit 58296888491e6c6bc8b1f79c45150888d6d722c4
Author:     Ian Jordan <immoloism@gmail.com>
AuthorDate: Sat Apr 6 10:09:03 2024
Commit:     Arthur Zamarin <arthurzam@gentoo.org>
CommitDate: Tue Apr 9 13:19:08 2024

    mate-base/mate-panel: add 1.28.0
    
    Signed-off-by: Ian Jordan <immoloism@gmail.com>
    Signed-off-by: Arthur Zamarin <arthurzam@gentoo.org>

diff --git a/mate-base/mate-panel/Manifest b/mate-base/mate-panel/Manifest
index b5b49a453e65..8a932592abbe 100644
--- a/mate-base/mate-panel/Manifest
+++ b/mate-base/mate-panel/Manifest
@@ -1 +1,2 @@
 DIST mate-panel-1.26.4.tar.xz 3262004 BLAKE2B 6561bdfe79f8b9f92db7fac1b3857bd0fb49b5220910f0aebc6eff443233c8cf2bdee8d21ffbdf88869ef1a99b3ffb96e0e22027099ef59d595ff596d79efdc3 SHA512 90db23f46dab88cb242aafb32737519d16e035843c2ca81da131ff3134a61b08772f19aa9900dbfbd00dcf4579cee3970765aa3fb461664abd1dd3babb8f6ea2
+DIST mate-panel-1.28.0.tar.xz 3282036 BLAKE2B 141dd58441b4dadd78e10a27e682b2a306e408151db50f9a57d2d480f74ab84f68efee16ead018c4185723d1af5b1abdf030a7841c0c522a7ae4e5b8377b4d8e SHA512 ef68fed6c9c60f7da11b5606e0b2669015f7f823ba392393c6e5d0705fe14ee21ffd13194879628cafacd319b83009d45db5949d2a42b9f5b07e10aeb6f8b308
diff --git a/mate-base/mate-panel/mate-panel-1.26.4.ebuild b/mate-base/mate-panel/mate-panel-1.28.0.ebuild
similarity index 91%
copy from mate-base/mate-panel/mate-panel-1.26.4.ebuild
copy to mate-base/mate-panel/mate-panel-1.28.0.ebuild
index 81061647c238..52a917988a26 100644
--- a/mate-base/mate-panel/mate-panel-1.26.4.ebuild
+++ b/mate-base/mate-panel/mate-panel-1.28.0.ebuild
@@ -7,8 +7,9 @@ MATE_LA_PUNT="yes"
 
 inherit mate
 
-if [[ "${PV}" != *9999 ]]; then
-	KEYWORDS="amd64 ~arm ~arm64 ~loong ~riscv x86"
+MINOR=$(($(ver_cut 2) % 2))
+if [[ ${MINOR} -eq 0 ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
 fi
 
 DESCRIPTION="The MATE panel"
@@ -25,6 +26,7 @@ COMMON_DEPEND="
 	>=dev-libs/libmateweather-1.17.0
 	dev-libs/libxml2:2
 	>=gnome-base/dconf-0.13.4
+	>=gnome-base/dconf-editor-43.0
 	>=mate-base/mate-desktop-1.17.0
 	>=mate-base/mate-menus-1.21.0
 	>=sys-apps/dbus-1.1.2
```

Bit late... :)

Anyway, if I understand correctly the reason this existing logic was already merged is because the very intentional avoidance of 9999 allows packaging odd-numbered releases which are considered unstable development versions that should not be keyworded. It's a common gnome ecosystem tactic, odd/even numbered releases.

Reply:
...but if they ever release a 9999 version, which will take them some time, we can move -live ebuilds to 99999999?

Reply:
@arthurzam ^

Reply:
I think you're missing the point which is that when packaging MATE 1.29.0 it is desirable to not have keywords despite building from an ordinary tarball fetched from a tagged release.

Reply:
(In fact, there are no live ebuilds built from git at all, and this ebuild doesn't conditionally inherit from git-r3 in an else branch.)

Reply:
The `$(( .... ))` construct is not a subshell.

Reply:
mkay. single ver_cut ain't so bad. leaves the unnecessary variable. ;)

Reply:
just add here, 9999 ebuilds are only used for testing fast moving packages such as mate-base/mate-wayland-session but in those cases I don't allow them into :gentoo and only in my mate-overlay. 

This is similar to what the Xfce team do but they mask 4.19 where as I prefer this method as it feels more like an extension to the git method whilst match Gentoo and upstream's wishes.

Happy to take suggestions on a better way to do though.